### PR TITLE
535 configure prettier for the project

### PR DESCRIPTION
**Pull request by EVeskje**
Configure prettier for the project. Added prettier to scripts (format).

### CODE REVIEW

**Review by IlyaAdlan — APPROVED** [review]
Prettier is added

File: .prettierrc.json
Comment (cnnrbrn) [inline .prettierrc.json:1]: singleQuote should be false, and tabWidth should be 2.

You can see how many files are in this PR due to the new formatting set here.

Comment (cnnrbrn) [reply .prettierrc.json:1]: After updating the config, create a new PR that only includes package.json and .prettierrc.json

Comment (EVeskje) [reply .prettierrc.json:1]: Changes is added :)

**Pull request closed without merging**